include/ui/console_multi_select_view.hpp

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <vector>
+#include <switch.h>
+
+#include "console_options_view.hpp"
+
+namespace tin::ui
+{
+    class ConsoleMultiSelectView : public ConsoleOptionsView
+    {
+        public:
+            ConsoleMultiSelectView(std::function<void ()> onDone = nullptr, std::string title="Tinfoil 0.1.0 by Adubbz", unsigned int unwindDistance = 1);
+
+            std::vector<ConsoleEntry*> GetSelectedEntries();
+            std::vector<IOptionValue*> GetSelectedOptionValues();
+
+            std::function<void ()> onDone;
+
+            void ProcessInput(u64 keys) override;
+        protected:
+            const char* PADDING_AFTER_CURSOR() const override
```

Comment:
I know this is a const function, however I think I'd still probably prefer it to be something like GetPaddingAfterCursor. UPPERCASE_WITH_UNDERSCORES stylistically seems best suited for enums, defines and static const stuff. Happy to hear your thoughts though if you disagree.

Reply:
Yeah, I initially had this as a `static const` and I forgot to change it when I changed it to a const function.